Math/is_power_of_two.js

```diff
@@ -0,0 +1,24 @@
+// Whether the provided number is power of two
```

Comment:
This program looks good, but there is O(log n ) and O(1) solution as well, could you look into this? https://www.geeksforgeeks.org/program-to-find-whether-a-given-number-is-power-of-2/****